go/logic/applier.go

```diff
@@ -1150,7 +1151,7 @@ func (this *Applier) ApplyDMLEventQueries(dmlEvents [](*binlog.BinlogDMLEvent))
 			return err
 		}
 
-		sessionQuery := "SET SESSION time_zone = '+00:00'"
+		sessionQuery := "SET /* gh-ost */ SESSION time_zone = '+00:00'"
```

Comment:
@timvaillancourt Does the time zone set here need to be configured as this.migrationContext.ApplierTimeZone? I noticed that other sessions are using this.migrationContext.ApplierTimeZone for setting the time zone, but here it is set to +00:00. Should this be adjusted?